examples/scenarios/windows_events.md

```diff
@@ -0,0 +1,55 @@
+# Windows Events
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure Windows Event monitoring via Stanza.
+
+## Prerequisites
+
+### Custom Channels
+
+The Windows Events plugin can monitor custom channels in addition to the three configured by default. In order to add a custom channel, you need the name of it which can be found in the left sidebar of the Windows Event Viewer window.
+
+## Configuration
+
+This is an example config file that can be used in the Stanza install directory. The Windows Events plugin supports system, application, and security events by default, but can also support custom channels if those have been configured.
+
+```yaml
+pipeline:
+  # To see the Windows Events plugin, go to: https://github.com/observIQ/stanza-plugins/blob/master/plugins/windows_event.yaml
+  - type: windows_event
+    enable_system_events: true
+    enable_application_events: true
+    enable_security_events: true
+    enable_custom_channels: false
```

Comment:
This is false by default, and can probably be omitted from the example.

Reply:
agreed. Was originally there from before I split off the custom events config example.